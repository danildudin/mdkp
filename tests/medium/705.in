30 5 0
123 388 433 882 306 854 639 169 960 607 494 989 24 518 925 30 907 152 169 491 103 963 319 781 435 286 867 674 38 998 
757 865 505 334 744 746 887 608 520 346 589 976 205 817 372 436 975 19 524 828 449 427 273 598 620 349 921 527 627 693 
861 834 879 567 685 48 356 57 891 5 282 339 809 412 36 412 294 331 989 656 793 362 704 942 987 757 527 909 800 326 
575 280 62 512 75 250 722 953 447 467 375 744 468 442 59 238 876 876 488 787 756 617 507 240 990 413 120 300 480 454 
563 917 555 43 692 393 851 940 763 666 951 820 541 837 558 847 359 594 703 396 372 63 969 553 609 651 378 351 584 700 
809 706 614 221 963 137 2 932 710 403 983 879 238 729 188 978 220 497 946 297 777 810 612 615 960 466 339 629 74 367 
5164 8999 7157 9022 5390 
