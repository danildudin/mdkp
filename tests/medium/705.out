8229
11
2 3 5 8 9 14 17 21 23 27 29 
